logging/access.go

```diff
@@ -44,6 +44,9 @@ type AccessEntry struct {
 
 	// The time that the request was received.
 	RequestTime time.Time
+
+	// The id of the authenticated user
+	AuthUser *string
```

Comment:
So you mean storing an empty string when the key is not present in the map? I basically added the indirection to avoid having to do that, but I can of course remove it if that's more in line with common practices here.

Reply:
Yes, empty string and `omitWhitespace` takes care to swap it for `"-"`